### PR TITLE
Add new mission form

### PR DESCRIPTION
**Pull request by gmassanek**
- Added some sub-components to make form creation a little simpler.
- Working mission form that redirects to index

### CODE REVIEW

File: tests/acceptance/user-can-create-mission-test.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import Ember from 'ember';
+import { module, test } from 'qunit';
+import startApp from 'agile-direction-ui/tests/helpers/start-app';
+import Pretender from 'pretender';
+
+var application,
+  missionsData,
+  server;
+
+module('Acceptance | user can view missions', {
```
Comment (stevenweber) [inline tests/acceptance/user-can-create-mission-test.js]: "user can **create** missions"?

File: tests/acceptance/user-can-create-mission-test.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+});
+
+test('can submit the new mission form and then see that mission', function(assert) {
+  visit('/missions/new');
+
+  fillIn('input.mission-name', 'My new mission');
+  click('button.submit');
+
+  andThen(function() {
+    assert.equal(find(`.mission .title a`).text().trim(), 'foo');
```
Comment (stevenweber) [inline tests/acceptance/user-can-create-mission-test.js]: Am I reading this right: "Create a mission called 'my new mission' and assert a mission named 'foo' now exists"?

File: tests/unit/components/form-field-test.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { moduleForComponent, test } from 'ember-qunit';
+
+moduleForComponent('form-field', 'Unit | Component | form field', {
+  // Specify the other units that are required for this test
+  // needs: ['component:foo', 'helper:bar'],
+  unit: true
+});
+
+test('it renders', function(assert) {
```
Comment (stevenweber) [inline tests/unit/components/form-field-test.js]: Is this all boilerplate generation?

**Comment by gmassanek** [comment]
This is abandoned right?

**Comment by stevenweber** [comment]
Yeah.

**Pull request closed without merging**